addons/l10n_bg/data/tax_report.xml

```diff
@@ -224,7 +224,7 @@
             <field name="parent_id" ref="l10n_bg_tax_report_b"/>
             <field name="sequence">203000</field>
             <field name="code">BG_TR_33</field>
-            <field name="formula">(BG_TR_11 + BG_TR_12_1 + BG_TR_13)/(BG_TR_01 + BG_TR_17 + BG_TR_18 + BG_TR_19)</field>
+            <field name="formula">(BG_TR_11 + BG_TR_12_1 + BG_TR_13)/max(BG_TR_01 + BG_TR_17 + BG_TR_18 + BG_TR_19, 1)</field>
```

Comment:
Is it possible to have a non zero `BG_TR_11` but a division by 0 for instance?
Wouldn't it be safer to just output 0 when the divisor is 0?